Resources/doc/reference/advanced_usage.rst

```diff
@@ -11,30 +11,36 @@ tweak this behavior by ignoring patterns :
     - ``ignore_routes``         : based on the route name
     - ``ignore_uri_patterns``   : based on the an uri pattern
 
-.. code-block:: yaml
+.. configuration-block::
 
-    sonata_page:
-        ignore_route_patterns:
-            - (.*)admin(.*)   # ignore admin route, ie route containing 'admin'
-            - ^_(.*)          # ignore symfony routes
+    .. code-block:: yaml
 
-        ignore_routes:
-            - sonata_page_cache_esi
-            - sonata_page_cache_ssi
+        # app/config/config.yml
 
-        ignore_uri_patterns:
-            - ^/admin\/     # ignore admin route, ie route containing 'admin'
+        sonata_page:
+            ignore_route_patterns:
+                - (.*)admin(.*)   # ignore admin route, ie route containing 'admin'
```

Comment:
strictly speaking, `ie` should be spelled `i.e.` (https://en.wikipedia.org/wiki/List_of_Latin_phrases_(I)#id_est)